src/FSharp.Core/printf.fs

```diff
@@ -658,8 +658,28 @@ module internal PrintfImpl =
     
     /// Contains functions to handle left/right and no justification case for numbers
     module GenericNumber =
-
-        let isPositive (n: obj) = 
+        
+        let inline doubleIsPositive (n: double) =
```

Comment:
Can you just use `double.IsPositive(n)`, or does F# need to differ?

.NET Framework notably has different behavior than .NET Core, which is also intentional as .NET Framework has a much higher backwards compatibility bar